Util/IronPython/Lib/iptest/console_util.py

```diff
@@ -105,7 +105,7 @@ def EatToMarker(self, marker, readError=False):
     # Execute a single-line command, and return the output
     def ExecuteLine(self, line, readError=False):
         self.writer.Write(line+"\n")
-        return self.EatToPrompt(readError)[0:-2]
+        return self.EatToPrompt(readError).rstrip()
```

Comment:
Unrelated to this PR, but it looks like we have some duplication. Any idea why this file appears in two locations (Util\IronPython\Lib\iptest and Languages\IronPython\IronPython\Lib\iptest)?